agent/testflinger_agent/tests/test_agent.py

```diff
@@ -203,3 +206,16 @@ def test_recovery_failed(self, agent, requests_mock):
             assert agent.check_offline()
         if os.path.exists(OFFLINE_FILE):
             os.unlink(OFFLINE_FILE)
+
+    def test_post_agent_data(self, agent, requests_mock):
+        # Make sure we post the initial agent data
+        with patch.object(
+            testflinger_agent.client.TestflingerClient, "post_agent_data"
+        ) as mock_post_agent_data:
+            agent._post_initial_agent_data()
+            mock_post_agent_data.assert_called_with(
+                {
+                    "queues": self.config["job_queues"],
+                    "location": self.config["location"],
+                }
+            )
```

Comment:
are you using `requests_mock` fixture here?

Reply:
good catch! I've removed it now